packages/atomic/cypress/e2e/generated-answer.cypress.ts

```diff
@@ -102,6 +105,45 @@ describe('Generated Answer Test Suites', () => {
       });
     });
 
+    describe('feedback modal', () => {
```

Comment:
Good start but IMO we should also test that the proper analytics events are sent.